ibis/expr/analysis.py

```diff
@@ -896,7 +896,7 @@ def try_fusion(self, root):
         parent_op = self.parent.op()
         for val in resolved:
             # XXX
```

Comment:
can this `XXX` turn into a useful comment, or be removed?

Reply:
Will remove it.